backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/createProductionOrder.feature

```diff
@@ -39,47 +40,49 @@ Feature: create production order
-      | Identifier | M_Product_ID.Identifier | PP_Product_BOM_ID.Identifier | PP_Product_Planning_ID.Identifier | S_Resource_ID | QtyEntered | QtyToProcess | QtyProcessed | C_UOM_ID.X12DE355 | DatePromised         | DateStartSchedule    | Processed | IsClosed |
-      | ocP_1      | p_1                     | bom_1                        | ppln_1                            | 540006        | 10         | 0            | 10           | PCE               | 2021-04-16T21:00:00Z | 2021-04-16T21:00:00Z | true      | false    |
-    And after not more than 10s, PP_Orders are found
+    Then after not more than 30s, PP_Order_Candidates are found
+      | Identifier | Processed | M_Product_ID.Identifier | PP_Product_BOM_ID.Identifier | PP_Product_Planning_ID.Identifier | S_Resource_ID | QtyEntered | QtyToProcess | QtyProcessed | C_UOM_ID.X12DE355 | DatePromised         | DateStartSchedule    | IsClosed |
+      | ocP_1      | true      | p_1                     | bom_1                        | ppln_1                            | 540006        | 10         | 0            | 10           | PCE               | 2021-04-16T21:00:00Z | 2021-04-16T21:00:00Z | false    |
+    And after not more than 30s, PP_Orders are found
       | Identifier | M_Product_ID.Identifier | PP_Product_BOM_ID.Identifier | PP_Product_Planning_ID.Identifier | S_Resource_ID | QtyEntered | QtyOrdered | C_UOM_ID.X12DE355 | C_BPartner_ID.Identifier | DatePromised         |
       | ppo_1      | p_1                     | bom_1                        | ppln_1                            | 540006        | 10         | 10         | PCE               | endcustomer_2            | 2021-04-16T21:00:00Z |
-    And after not more than 10s, PP_Order_BomLines are found
+    And after not more than 30s, PP_Order_BomLines are found
       | PP_Order_ID.Identifier | Identifier | M_Product_ID.Identifier | QtyRequiered | IsQtyPercentage | C_UOM_ID.X12DE355 | ComponentType |
       | ppo_1                  | ppol_1     | p_2                     | 100          | false           | PCE               | CO            |
-    And after not more than 10s, PP_OrderCandidate_PP_Order are found
+    And after not more than 30s, PP_OrderCandidate_PP_Order are found
       | PP_Order_Candidate_ID.Identifier | PP_Order_ID.Identifier | QtyEntered | C_UOM_ID.X12DE355 |
       | ocP_1                            | ppo_1                  | 10         | PCE               |
-    And after not more than 10s, MD_Candidates are found
-      | Identifier | Type   | BusinessCase | M_Product_ID.Identifier | Time                 | DisplayQty | ATP |
-      | c_3        | SUPPLY | PRODUCTION   | p_1                     | 2021-04-16T21:00:00Z | 10         | 0   |
+    # this is mostly about de.metas.material.dispo.service.candidatechange.StockCandidateService.applyDeltaToMatchingLaterStockCandidates finishing work before checking the candidates
+    And wait for 30s
+    And after not more than 30s, MD_Candidates are found
```

Comment:
why hardcode the test to wait for 30s and then wait up to 30s more? i mean, if metasfresh sometimes really takes that long, why not have just `And after not more than 60s, MD_Candidates are found` ?

Reply:
Later edit: 
I noticed
`# this is mostly about de.metas.material.dispo.service.candidatechange.StockCandidateService.applyDeltaToMatchingLaterStockCandidates finishing work before checking the candidates`

if the we need a different solution.
The `after not more than 30s, MD_Candidates are found`  should wait for the other results, too. 
And/or maybe cucumber can wait for the handler to be finished (`wait/notify`, `CountDownLatch`)
But we can't delay the build for another 4 additional minutesin one feature file.